### DISCUSSION TITLE
response data  body  type array  is out of work

### DISCUSSION THREAD

**Discussion by markguy** [postmanlabs/postman-flows#397] [Q&A]
response  body  data type  is array ，but  cannot get variables like this “{{body[0].id}}”,  the console log is "https://www.gitlab.com/api/v4/users/%7B%7Bbody[0]%7D%7D"

**Comment by samay-v** [comment]
Are you set the variables from a test/prerequest script? Env are treated readonly by flows, here's why https://github.com/postmanlabs/postman-flows/discussions/142
